js/ui/windowManager.js

```diff
@@ -2285,8 +2656,8 @@ var WindowManager = class {
                     this._showOtherWindows(actor, false);
             }
 
-            /* If this is the Discovery Feed, notify it that it has
-             * finished closing now */
+            // If this is the Discovery Feed, notify it that it has
```

Comment:
nitpick: Rebase leftover? This change can be removed from this branch

Reply:
Fixed.